docs/source/user/interface.rst

```diff
@@ -88,13 +88,13 @@ The same information is also available in the Tabs menu:
 .. image:: images/interface_tabs_menu.png
    :align: center
 
-It is often useful to focus on a single document or activity without
-closing other tabs in the main area. Single-Document Mode toggles the
-view of the main area to show only a single tab at a time:
+It is often useful to focus on a single document or activity without closing
+other tabs in the main area. The Single-Document Mode item in the View menu
+toggles the view of the main area to show only a single tab at a time:
 
 [animation]
 
-When you leave Single-Document Mode, the original layout of the main
+When you leave single-document mode, the original layout of the main
```

Comment:
why is this in lower case if other menu labels are in title case?

Reply:
Here, I'm not referring to the menu item for them to click on, but I'm talking about the generic concept, so I think it should be lower-case.

Reply:
Are we making that distinction in the language that we settled on? I didn't realize we could swap in certain cases.

Reply:
I agree that the idea should be lowercase, but the menu item should be title case.